<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>order</class>
 <widget class="QDialog" name="order">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Drug Order</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #bfe0f4;</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Order ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Drug Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Company Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Supplier Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Cost Price</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Order Date</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_oid">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);

border-radius:15px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_oname">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ocmpname">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_osupname">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_oqnty">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ocst">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_od">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>821</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_oentry">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Order Entry</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_dentry">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Delete Order</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Drug Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_otype">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#3f76dc;
border-radius:10px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:10px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Display/Refresh table</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_search">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>71</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_search">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>70</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
